packages/terra-form-input/src/InputField.jsx

```diff
@@ -143,6 +149,12 @@ const InputField = (props) => {
     }
   }
 
+  let isDisabled;
```

Comment:
To remove the toggle below, should we update this to be? or should `disabled` should have higher precedence. What do others think? @StephenEsser opinions since you logged the issue?

```
let isDisabled = disabled;

if (inputAttrs.disabled} {
    isDisabled = true;
}

```

Reply:
Is there a reason we need to do the if check at all? Could just do:
`const isDisabled = inputAttrs.disabled || disabled;`
or remove this and below do:
`disabled={inputAttrs.disabled || disabled}`

Reply:
The second approach seems ideal to me.

```jsx
disabled={inputAttrs.disabled || disabled}
```

Reply:
I'll check it out. I toyed around with many ways of going about it and the way I have was the first I could get to work.